nexteuropa_newsroom.module

```diff
@@ -667,7 +667,7 @@ function _nexteuropa_nexteuropa_newsroom_item_importer($newsroom_id = 0, $delete
     // Prepare URLs for importers and get feed importer ID.
     $url = NewsroomHelper::getImporterUrl(NEXTEUROPA_NEWSROOM_ITEM_IMPORTER) . variable_get('newsroom_single_item_import_segment', NEXTEUROPA_NEWSROOM_ITEM_SEGMENT) . $newsroom_id;
     NewsroomHelper::runFeedImporter(NEXTEUROPA_NEWSROOM_ITEM_IMPORTER, $url);
-    drupal_set_message('The item has been imported');
+    drupal_set_message(t('The item has been imported'));
```

Comment:
Following the standard recommendation by api.drupal.org here https://api.drupal.org/api/drupal/includes%21bootstrap.inc/function/drupal_set_message/7.x please put there the full stop.